tests/train_vae_test.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    assert pd.read_csv(output_path, sep="\t")["val_loss"].values[-1] &lt; 2800, pd.read_csv(output_path, sep="\t")["val_loss"].values[-1]
</pre><p>just curious -- is this non-determinisitic? If its determinisitic, why not have == 2079.0769? Is that a rounded value or something?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/ponyo/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
The VAE training is non-deterministic due to the randomness set in the background of python. Originally we checked for exact matching as you described but this didn't work.